src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -638,6 +638,7 @@
             color: $gray-dark;
             // FIXME: this should be in the general reboot
             cursor: pointer;
+            outline: none;
```

Comment:
HI Aditya,
I am tested in chrome it is accessible via keyboard also. can you please test and let me know. If its not working. i will close the PR.
Thanks,
Srinivas P